libs/services/designer-client-services/src/lib/base/gateway.ts

```diff
@@ -29,7 +29,7 @@ export class BaseGatewayService implements IGatewayService {
   }
 
   public getGateways(subscriptionId: string, connectorName: string): Promise<Gateway[]> {
```

Comment:
`subscriptionId: string | undefined`

Reply:
This is the base implementation of getGateways. Since our service will have it's own implementation we should be able to leave this, right? The default method of grabbing gateways here relies on this parameter.

Reply:
The implementation should align with the interface it inherits from, IMO, because you could call the interface with `undefined` while the implementation is expecting a string. I'm not sure why TypeScript doesn't enforce this, but they've chosen not to.